src/store/cache/users/sagas.js

```diff
@@ -39,6 +42,19 @@ export function* fetchUsers(
   })
 }
 
+function* retrieveUserByHandle(handle) {
+  const userId = yield select(getUserId)
+  if (Array.isArray(handle)) {
+    handle = handle[0]
+  }
+  const encodedUserId = userId ? encodeHashId(userId) ?? undefined : undefined
```

Comment:
I think we moved this conversion to the apiClient

Reply:
thx good catch